package-lock.json

```diff
@@ -0,0 +1,13320 @@
+{
```

Comment:
this too?

Reply:
I didn't touch this file

Reply:
how do I resolve this file?

Reply:
Do a `-i`  rebase and select `edit` for the commit in which this got commited, and do not add this file (add all other files) and then `git rebase --continue`.

Let me know on chat.zulip.org if you face any problem. :)